amazon-s3/5.8/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -9,7 +9,8 @@ To add and configure a connector in Studio:
 . <<configure-source,Configure a source for the connector's flow>>.
 . <<add-connector-operation,Add a connector operation to the flow>>.
 . <<configure-global-element,Configure a global element for the connector>>.
-. <<configure-other-fields,Configure the Other Connector Fields>>.
+. <<view-app-log,View the app log>>.
```

Comment:
this one might not need to be added since view the app log is referenced below

Reply:
Oh, you're right!